### DISCUSSION TITLE
Wish: RoundAustin format for ExecutionTime

### DISCUSSION THREAD

**Discussion by JensGJ** [JanDeDobbeleer/oh-my-posh#3111] [Ideas]
I would love to have a style for execution time that rounded a little less that the round style does. The round style works fine for me with values less than one second (no rounding) and values larger than one minute (shown e.g. as 2m 4s). 
But for values between 1 and 59 seconds (perhaps especially between 1 and 10) I would prefer a fraction part - e.g. 2.4 seconds (rounded to precision 1) or just 2.461 s (no rounding, similar to the Austin format).

**Comment by JanDeDobbeleer** [comment]
@JensGJ can you a few use-cases I can use to create some tests?

What I would do:

```
if < 1 second
24ms
if < 1 minute
2.4s
if < 1 hour
2.4m
if < 1 day
2.4h
else
2.4d
```
